content/tracing/advanced_usage/distributed_tracing.md

```diff
@@ -75,6 +75,24 @@ public class MyHttpRequestExtractAdapter implements TextMap {
 
 {{% /tab %}}
 {{% tab "Python" %}}
+Distributed tracing is supported in the following frameworks. Please refer to
+the configuration documentation for each to enable it.
+
+| Framework/Library | Default  |                          API Documentation                          |
+| ----------------- | :------- | :------------------------------------------------------------------ |
+| aiohttp           | Disabled | http://pypi.datadoghq.com/trace/docs/web_integrations.html#aiohttp  |
+| bottle            | Enabled  | http://pypi.datadoghq.com/trace/docs/web_integrations.html#bottle   |
```

Comment:
Same here: https://github.com/DataDog/dd-trace-py/blob/master/ddtrace/contrib/bottle/trace.py#L15

Reply:
I think you can remove the column at this point, since everything is disabled by default.

Reply:
yep